libhtmlw/HTMLparse.c

```diff
@@ -111,7 +111,37 @@ unsigned char map_table[256]={
 #define TOLOWER(x)	(map_table[x])
 #endif /* NOT_ASCII */
 
-
+/* Converts an UCS code < 65536 into a UTF-8 string. Returns the string length */  
+int ucs2utf8(unsigned int ucs,char code[4])
```

Comment:
Please use same indentation style as remaining code.

Reply:
That's slightly better now, but the second level of indentation is still in GNU style, not the project's style.  All indentation should be one tab per level and no “braces at half the indentation” stuff.